src/schedule-and-details/hooks.jsx

```diff
@@ -79,6 +84,15 @@ const useSaveValuesPrompt = (
       setTimeout(() => setShowSuccessfulAlert(false), 15000);
       window.scrollTo({ top: 0, behavior: 'smooth' });
 
+      if (!isEditableState) {
+        setShowModifiedAlert(false);
+      }
+    } else if (savingStatus === RequestStatus.FAILED) {
+      setIsQueryPending(false);
+      setShowFailedAlert(true);
+      setTimeout(() => setShowFailedAlert(false), 15000);
+      window.scrollTo({ top: 0, behavior: 'smooth' });
```

Comment:
Should we also undo the changes that they made?

Reply:
Do you mean the changes to dates/times that the user makes? I think we should keep those. If the connection failure isn't due to the backend erroring, then the user can try to submit again.